### PR TITLE
STAC-21851: Document new otel http ingress. and that we have that exp…

### PR DESCRIPTION
**Pull request by aacevedoosorio**
…orter available

### CODE REVIEW

File: setup/otel/collector.md [hunk 1]
```diff
@@ -68,6 +68,10 @@ config:
       auth:
         authenticator: bearertokenauth
       endpoint: <otlp-stackstate-endpoint>:443
+    otlphttp/stackstate:
+       auth:
+          authenticator: bearertokenauth
+       endpoint: https://<otlp-stackstate-endpoint>
```
Comment (rb3ckers) [inline setup/otel/collector.md]: `otlp-stackstate` -> `otlp-http-stackstate` to make clear it is a different endpoint

File: use/security/k8s-ingestion-api-keys.md [hunk 2]
```diff
@@ -104,4 +104,8 @@ The following code snippet provides an example configuration:
       auth:
         authenticator: bearertokenauth
       endpoint: <otlp-stackstate-endpoint>:443
+    otlphttp/stackstate:
+      auth:
+        authenticator: bearertokenauth
+      endpoint: https://<otlp-stackstate-endpoint>
```
Comment (rb3ckers) [inline use/security/k8s-ingestion-api-keys.md]: otlp-stackstate -> otlp-http-stackstate to make clear it is a different endpoint

File: setup/otel/collector.md [hunk 3]
```diff
@@ -34,7 +34,7 @@ To install and configure the collector for usage with SUSE Observability we'll u
 ### Configure the collector
 
 Here is the full values file needed, continue reading below the file for an explanation of the different parts. Or skip ahead to the next step, but make sure to replace:
-* `<otlp-stackstate-endpoint>` with the OTLP endpoint of your SUSE Observability. If, for example, you access SUSE Observability on `play.stackstate.com` the OTLP endpoint is `otlp-play.stackstate.com`. So simply prefixing `otlp-` to the normal SUSE Observability url will do.
+* `<otlp-stackstate-endpoint>` with the OTLP endpoint of your SUSE Observability. If, for example, you access SUSE Observability on `play.stackstate.com` the OTLP endpoint is `otlp-play.stackstate.com` for GRPC and `otlp-http-play.stackstate.com` for  HTTP traffic. So simply prefixing `otlp-` ot `otlp-http-` to the normal SUSE Observability url will do.
```
Comment (rb3ckers) [inline setup/otel/collector.md]: Typo: `ot` -> `or`.

File: setup/otel/collector.md (hunk 1 above)
Comment (aacevedoosorio) [reply setup/otel/collector.md]: applied the change

File: use/security/k8s-ingestion-api-keys.md (hunk 2 above)
Comment (aacevedoosorio) [reply use/security/k8s-ingestion-api-keys.md]: applied the change

File: setup/otel/collector.md (hunk 3 above)
Comment (aacevedoosorio) [reply setup/otel/collector.md]: applied the change

**Pull request merged**